cmd/oras/internal/display/track/target.go

```diff
@@ -66,6 +67,14 @@ func NewTarget(t oras.GraphTarget, actionPrompt, donePrompt string, tty *os.File
 	return gt, nil
 }
 
+// Mount makes graphTarget implement oras.Mounter.
+func (t *graphTarget) Mount(ctx context.Context, desc ocispec.Descriptor, fromRepo string, getContent func() (io.ReadCloser, error)) error {
+	if mounter, ok := t.GraphTarget.(registry.Mounter); ok {
+		return mounter.Mount(ctx, desc, fromRepo, getContent)
+	}
+	return errors.New("not implemented")
+}
```

Comment:
Do we need a new structure to make OCI layout scenarios work?

Reply:
OCI layout will be blocked by `opts.MountFrom`

Reply:
Can you elaborate more about `work`? If there is an OCI layout involved in copying, we only need to block this method from being called. Currently it's via `opts.MountFrom`.

Reply:
Discussed offline with @shizhMSFT, will update to type assertion.